convert/src/vertex_to_bd/labelstudio_format/labelstudio_format.py

```diff
@@ -0,0 +1,63 @@
+import uuid
+from pathlib import Path
+from typing import List
+
+from ..models.bd_annotation_model import BadgerdocAnnotation, Obj
+from ..models.bd_tokens_model import BadgerdocToken, Page
+from ..models.vertex_models import (
+    Annotation,
+    Data,
+    LabelStudioModel,
+    ModelItem,
+    ResultItem,
+    Value,
+)
+
+
+class LabelStudioFormat:
+    # TODO: support more than 1 page
+    def __init__(self) -> None:
+        self.labelstudio_data = LabelStudioModel()
+
+    def from_badgerdoc(
+        self,
+        badgerdoc_tokens: Page,
+        badgerdoc_annotations: BadgerdocAnnotation,
+    ):
+        text = "".join([obj.text for obj in badgerdoc_tokens.objs])
+        result_items = self.convert_annotation_from_bd(
+            badgerdoc_annotations, badgerdoc_tokens.objs
+        )
+        item = ModelItem(
+            annotations=[Annotation(result=result_items)],
+            predictions=[],
+            data=Data(text=text),
+        )
+        self.labelstudio_data.__root__.append(item)
+
+    def convert_annotation_from_bd(
+        self, annotations: BadgerdocAnnotation, tokens: List[BadgerdocToken]
+    ):
+        objs = annotations.pages[0].objs
+        result_items = []
+        for obj in objs:
+            if not obj.tokens:
+                continue
+            item = ResultItem(
+                id=str(uuid.uuid4())[:10],
```

Comment:
There is a field data in badgerdoc annotations format with this id (optional)
`obj.data["source_id"] `